cmake/tribits/ReleaseNotes.txt

```diff
@@ -2,6 +2,15 @@
 Release Notes for TriBITS
 ----------------------------------------
 
+2020/06/16:
+
+(*) MINOR: The variables <Package>_FORTRAN_COMPILER and
+    <Package>_FORTRAN_FLAGS in the <Package>Config.cmake files (in the build
+    dir and the install dir) are deprecated.  Plese use the new varibles
```

Comment:
typo: "Please", "variables"

Reply:
Typo fixed in https://github.com/TriBITSPub/TriBITS/commit/f6c9f1ae9065dedc576d6890ec95aa1466a1815b.